server/data/migrations/dev/20190214183841_refreshrs_table.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function(knex, Promise) {
+	return knex.schema.createTable('refreshrs', (tbl) => {
+		tbl.increments();
+		tbl.date('date');
+		tbl
+			.integer('class_id')
+			.unsigned()
```

Comment:
Is this best practice? I don't think I added unsigned to the foreign keys in my migrations :fearful:

Reply:
Yeah it's to guard against negative numbers, which helps increase the range of numbers by pushing out unused negative number and allowing more positive numbers. Here is a good explanation of it: https://stackoverflow.com/questions/11515594/when-should-i-use-unsigned-and-signed-int-in-mysql/11515613#11515613

Reply:
Thank you! Makes perfect sense.